supertokens_python/supertokens.py

```diff
@@ -211,6 +216,69 @@ def get_all_cors_headers(self) -> List[str]:
 
         return list(headers_set)
 
+    async def get_user_count(self, include_recipe_ids: List[str] = None) -> int:
+        querier = Querier.get_instance(None)
+        api_version = querier.get_api_version()
```

Comment:
`get_api_version` is an async function and you need to await the result. Also we don't support api-version prior to 2.9 so you can remove the api-version checking part